### PR TITLE
Create Pull Request

### PR DESCRIPTION
**Pull request by kylie-brothers**

### CODE REVIEW

File: ggplot2_matplotlib_cheatsheet.Rmd [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Data Visualization with GGPlot2 (R) and MatPlotLib (Python) Cheat Sheet
+
+Motivation
```
Comment (philkim72) [inline ggplot2_matplotlib_cheatsheet.Rmd:3]: The second line should be blank, followed by the author name(s).

**Review by philkim72 — CHANGES_REQUESTED** [review]
The second line should be blank, followed by the author name(s).

File: ggplot2_matplotlib_cheatsheet.Rmd (hunk 1 above)
Comment (kylie-brothers) [reply ggplot2_matplotlib_cheatsheet.Rmd:5]: I have added the author name to the 3rd line of code

**Comment by philkim72** [comment]
Thank you for your contribution! :tada:

**Pull request merged**